### PR TITLE
Improve v4 release process with grabthar-release

### PR DESCRIPTION
**Pull request by nbierdeman**

### CODE REVIEW

**Comment by nbierdeman** [comment]
@bluepnume Mind checking this out when you've got a sec?

File: scripts/postversion.sh [hunk 1]
```diff
@@ -5,3 +5,12 @@ git push;
 git push --tags;
 git push git@github.com:paypal/paypal-checkout.git v4;
 npm publish;
+
+# verify new version has been published and then publish to cdnx
```
Comment (gregjopa) [inline scripts/postversion.sh:9]: One thought around improving this comment:
```suggestion
# publish to cdnx after verifying a successful publish to the public npm registry
```

**Review by bluepnume — APPROVED** [review]
Looks great to me! Thanks for taking this up Nate!

File: scripts/postversion.sh (hunk 1 above)
Comment (nbierdeman) [reply scripts/postversion.sh]: Updated. Thanks for the suggestion!

**Review by gregjopa — APPROVED** [review]
Awesome work @nbierdeman! 💯 🎉 🥇

**Comment by nbierdeman** [comment]
@bluepnume @gregjopa Do either of you have ideas/suggestions for getting the CI to build correctly? I was hoping it was just a fluke, and clicked the 'rerun' button a couple of times yesterday, to no avail.

**Pull request merged**